UnityProject/Assets/Scripts/Chemistry/ChemistryInitializationTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class ChemistryInitializationTest : MonoBehaviour {
+
+		// Use this for initialization
+		void Awake () {
+			Chemistry.Initialization.run();
+			ChemistryTest();
+		}
+
+
+	
+		public static void ChemistryTest()
+		{
+			var List_of_reactions = Chemistry.Globals.List_of_reactions;
```

Comment:
could you please remove underscores from variables and method names?
also they could be shorter, like `List_of_reactions` could be just `reactions`, or `reactionList` if you're insistent